### PR TITLE
Updates to GH readme content

### PR DESCRIPTION
**Pull request by Jen-OMalley**
The PR addresses [this issue](https://github.com/AlexsLemonade/AdminItems/issues/967) filed in `admin`.

- Updated ScPCA content and added community contributions info
- Changed the OpenPBTA content to past tense and linked the paper
- Added external training workshops content
- Added LinkedIn

Should the `.DS_store` file be deleted? Will do that, if needed!

.

### CODE REVIEW

**Comment by jaclyn-taroni** [comment]
> Should the `.DS_store` file be deleted? Will do that, if needed!

Yep. `.DS_Store` files are just for your Mac operating system. You'll never need to commit them. 

I'd recommend creating a `.gitignore` in your home directory or the root of wherever Git repos live on your computer that includes `.DS_Store` files. That will ignore them everywhere you'd potentially have the _opportunity_ to commit them. I'm happy to walk you through this suggestion.

File: profile/README.md [hunk 1]
```diff
@@ -23,34 +23,53 @@ The Data Lab team simultaneously contributes to childhood cancer research and to
...
+
 ## Open Pediatric Brain Tumor Atlas (OpenPBTA)
 
-[`Open Pediatric Brain Tumor Atlas (OpenPBTA)`](https://www.ccdatalab.org/openpbta) is a global open science initiative, which analyzes a vast collection of pediatric brain tumor data, comprising data from 943 tumors. 
-This project operates on an open contribution model, crowdsourcing expertise from childhood brain cancer experts from across the world. 
+The [`Open Pediatric Brain Tumor Atlas (OpenPBTA)`](https://www.ccdatalab.org/openpbta) project was a global open science initiative, which analyzed a vast collection of pediatric brain tumor data, comprising data from over 1,000 tumors. 
+This project operated on an open contribution model, crowdsourcing expertise from childhood brain cancer experts from across the world. 
+
+In May 2023, this work was published in *Cell Genomics*. 
+[Read the OpenPBTA paper](https://www.cell.com/cell-genomics/fulltext/S2666-979X(23)00115-5) to learn more!
```
Comment (jaclyn-taroni) [inline profile/README.md]:
```suggestion
[Read the OpenPBTA paper](https://doi.org/10.1016/j.xgen.2023.100340) in *Cell Genomics* to learn more!
```

File: profile/README.md [hunk 2]
```diff
@@ -23,34 +23,53 @@ The Data Lab team simultaneously contributes to childhood cancer research and to
...
 
-* [See the collaborative analysis in real-time.](https://github.com/AlexsLemonade/OpenPBTA-analysis)
-* [View the collaboratively written manuscript.](https://github.com/AlexsLemonade/OpenPBTA-manuscript)
+* [View the analysis repository.](https://github.com/AlexsLemonade/OpenPBTA-analysis) (now archived)
+* [View the repository for the collaboratively written manuscript.](https://github.com/AlexsLemonade/OpenPBTA-manuscript) (now archived)
 
 # Training Workshops
 
-We offer [`training workshops`](http://ccdatalab.org/training) to teach researchers the data science skills they need to examine their own data. Participants are introduced to the R programming language and to cutting-edge technologies used in single-cell and bulk RNA-sequencing data analysis. All of our training materials are openly licensed and freely available!
+We offer [`training workshops`](http://ccdatalab.org/training) to teach researchers the data science skills they need to examine their own data.
```
Comment (jaclyn-taroni) [inline profile/README.md]:
```suggestion
We offer [`training workshops`](http://ccdatalab.org/training) to teach pediatric cancer researchers the data science skills they need to examine their own data.
```

File: profile/README.md [hunk 3]
```diff
@@ -23,34 +23,53 @@ The Data Lab team simultaneously contributes to childhood cancer research and to
...
+All Data Lab training materials are openly licensed and freely available for others to use. 
+**Interested in using our materials to hold your own workshop?** 
+[Learn how to get started](https://www.ccdatalab.org/hold-a-workshop) and [fill out the instructor interest form](https://share.hsforms.com/1KgR746BzThGUs-NG4oHgsw336z0) to submit an inquiry.
+
+Email [`training@ccdatalab.org`](mailto:training@ccdatalab.org) with any questions about attending or holding a workshop.
 
 # Get Involved
 
-Visit us at [`ccdatalab.org`](http://ccdatalab.org) and follow us on Twitter, [`@CancerDataLab.`](https://twitter.com/cancerdatalab)
+Visit us at [`ccdatalab.org`](http://ccdatalab.org), follow us on X at [`@CancerDataLab`](https://twitter.com/cancerdatalab), and connect with us on [`LinkedIn`](https://www.linkedin.com/showcase/childhood-cancer-data-lab/).
```
Comment (jaclyn-taroni) [inline profile/README.md:71]: Okay, but do we have to call it X?

**Comment by Jen-OMalley** [comment]
That would be great @jaclyn-taroni! If you'd like, I can delete it from this PR for now, and maybe you can show me how to create a `.gitignore` during our next comm meeting or 1:1?

File: .DS_Store
Comment (jaclyn-taroni) [inline .DS_Store:1]: To answer your question about deleting this for now – yes, you will need to delete this, no matter what. Adding a `.gitignore` as described would only prevent you from committing further changes to this file (in the context of this repository).

File: profile/README.md [hunk 4]
```diff
@@ -23,34 +23,53 @@ The Data Lab team simultaneously contributes to childhood cancer research and to
 
 ## Single-cell Pediatric Cancer Atlas (ScPCA)
 
-[`The Single-cell Pediatric Cancer Atlas (ScPCA)`](https://scpca.alexslemonade.org/) focuses on single-cell and single-nuclei profiling with the goal of creating a publicly available atlas of pediatric cancer data. 
-Ten ScPCA awards were funded by Alex’s Lemonade Stand Foundation. 
-The ALSF-funded researchers submit their single-cell, single-nuclei, and bulk RNA sequencing data to the Data Lab for processing. 
-The data from these patient tumors are made available in one location through the [ScPCA Portal](https://scpca.alexslemonade.org/).
+ALSF created the [`Single-cell Pediatric Cancer Atlas (ScPCA)`](https://scpca.alexslemonade.org/) project to generate an unprecedented resource for the pediatric cancer research community.
```
Comment (jaclyn-taroni) [inline profile/README.md]: This applies elsewhere, too – I don't think these project names need to be in back ticks. I know this is not new but is something I just noticed looking at the rich diff.

```suggestion
ALSF created the [Single-cell Pediatric Cancer Atlas (ScPCA)](https://scpca.alexslemonade.org/) project to generate an unprecedented resource for the pediatric cancer research community. 
```

File: profile/README.md [hunk 5]
```diff
@@ -23,34 +23,53 @@ The Data Lab team simultaneously contributes to childhood cancer research and to
...
+* [View the full contribution guidelines](https://scpca.alexslemonade.org/contribute) to learn more.
+* [Submit an interest form](https://share.hsforms.com/1Fs_Z94OzS7abnh5qEieOTA336z0) to be notified about our next call for contributions!
+
+Email [`scpca@ccdatalab.org`](mailto:scpca@ccdatalab.org) with any questions about the Portal or submitting data.
+
 ## Open Pediatric Brain Tumor Atlas (OpenPBTA)
 
-[`Open Pediatric Brain Tumor Atlas (OpenPBTA)`](https://www.ccdatalab.org/openpbta) is a global open science initiative, which analyzes a vast collection of pediatric brain tumor data, comprising data from 943 tumors. 
-This project operates on an open contribution model, crowdsourcing expertise from childhood brain cancer experts from across the world. 
+The [`Open Pediatric Brain Tumor Atlas (OpenPBTA)`](https://www.ccdatalab.org/openpbta) project was a global open science initiative, which analyzed a vast collection of pediatric brain tumor data, comprising data from over 1,000 tumors.
```
Comment (jaclyn-taroni) [inline profile/README.md]:
```suggestion
The [Open Pediatric Brain Tumor Atlas (OpenPBTA)](https://www.ccdatalab.org/openpbta) project was a global open science initiative, which analyzed a vast collection of pediatric brain tumor data, comprising data from over 1,000 tumors. 
```

File: profile/README.md (hunk 4 above)
Comment (jaclyn-taroni) [reply profile/README.md]: I can not easily comment on the refine.bio instance of this pattern.

**Review by jaclyn-taroni — APPROVED** [review]
Looks good, with a few edits (and a somewhat unserious comment at line 72)

**Pull request merged**